### ISSUE TITLE
Error creating schedule for job template

### ISSUE THREAD

**Issue by joshiparth1000** [denouche/terraform-provider-awx#34]
I tried creating a schedule for a job template but got the following error:

resource "awx_schedule" "backup" {
Schedule failed to create Errors:
- inventory: [Invalid pk "0" - object does not exist.]

How can I further troubleshoot this issue?

**Comment by fernando-eickhoff** [comment]
Same error here:

Resource creation:
```
resource "awx_schedule" "solr_staging_schedule" {
    name                      = "solr-staging-schedule"
    rrule                     = "DTSTART;TZID=Europe/Paris:20211214T120000 RRULE:INTERVAL=1;FREQ=DAILY"
    unified_job_template_id   = awx_job_template.solr_cluster_job_template_staging.id
}
```
Output:
```
│ Error: Unable to create Schedule
│ 
│   with awx_schedule.solr_staging_schedule,
│   on main.tf line 480, in resource "awx_schedule" "solr_staging_schedule":
│  480: resource "awx_schedule" "solr_staging_schedule" {
│ 
│ Schedule failed to create Errors:
│ - inventory: [Invalid pk "0" - object does not exist.]
```

**Comment by spacex** [comment]
This appears to be similar to #19

**Comment by gfa** [comment]
v0.26.0 releases solves this problem (tested with AWX 24.0.0)

@denouche can you pretty please release it into registry.terraform.io? it would make it a lot easier to use it

thanks!

**Comment by spacex** [comment]
I can confirm the issues are resolved in v0.27.0. Thanks!
